react/src/screens/BookDetail/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import BookDetailCommentaries from '../../components/BookDetailCommentaries';
+import BookDetailData from '../../components/BookDetailData';
+import BookDetailSuggestions from '../../components/BookDetailSuggestions';
+import { NavLink } from 'react-router-dom';
+import './styles.css';
+
+class BookDetail extends Component {
+  render() {
+    return (
+      <div>
+        <NavLink to="/" className="go-back">
```

Comment:
Use route constants here

Reply:
It will be interesting to implement later a `go-back` function here, so if we clicked a suggestion from a book detail, we don't go to HOME but to the previous book detail. I'd keep this as a nice to have.